play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukSelect.scala.html

```diff
@@ -46,9 +46,9 @@
           @defining((value.exists(_.nonEmpty) && item.value == value) || item.selected) { isSelected =>
             @* Allow selecting by text content (the value for an option when no value attribute is specified) *@
             @defining(item.value.getOrElse(item.text)) { effectiveValue =>
-              <option @if(item.value.nonEmpty) {value="@item.value"}
+              @if(item.value.nonEmpty || isSelected || item.disabled || !item.attributes.isEmpty || item.text != ""){<option @if(item.value.nonEmpty) {value="@item.value"}
```

Comment:
same for this one, is it recreating this

https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/select/template.njk#L51

so allowing falsey items - like if someone passed `items = [ false, undefined, { .. actual item ... } ]`?

I think for these, like with the accordion we can exclude support for these cases, because people will be prevented by the type system from entering nonsense so we don't need to account for it

Reply:
Just had a dig through and tested the previous one and with this - And I think you've got it. I believe these were failing the fixtures, the changes I proposed "fixed" those issues and then when we uncovered the falsey -> falsy name change, I updated the excluded-fixtures to the correct names. 

I think these fixtures are now not being checked anyway (At least it's not failing any more with these changes removed). Will commit the removal of the changes for checkboxes and select 🗑️

Reply:
sweet 🙌